### ISSUE TITLE
Command-line tool won’t update the `qs` command

### ISSUE THREAD

**Issue by skurfer** [quicksilver/Quicksilver#2264]
It warns you that it needs to be updated, but hitting the button doesn’t actually overwrite `/usr/local/bin/qs`, so the warning stays in place.

Also, the text of the alert says the command is missing. It should be updated to say “missing or outdated”. Or really, if the file already exists, we should feel free to replace it silently (once that’s working).

**Comment by tiennou** [comment]
I think `/usr/local/` is not writable by default (unless you use `homebrew`), so maybe there's something funky going on with permission.

It also doesn't log anything that could help to pinpoint the issue, right :trollface: ?

**Comment by skurfer** [comment]
> I think `/usr/local/` is not writable by default

That’s an issue when we first install, which has already been dealt with (I think). The problem I saw was that the command never got replaced, even though I have write access to that file.

My guess is that something wants confirmation before replacing a file, and it’s silently rejected in the background.

**Comment by natenate19** [comment]
I ran into the same issue.  I just moved `/usr/local/bin/qs` out of the way:

```
samsara:~ nate$ mv /usr/local/bin/qs .
```

Restarted Quicksliver, installed the Command Line Tool afresh, and it installed fine.  The warning is resolved.  Looking at the original `/usr/local/bin/qs` it looked like this:

```
-rwxr-xr-x 1 nate admin 30K Oct  1  2015 /usr/local/bin/qs
```

The new one has slightly different group ownership:

```
-rwxr-xr-x 1 nate staff 26K Aug 19 10:43 /usr/local/bin/qs
```

Perhaps an issue from the El Capitan rootless / SIP migration.  This is all on OS X 10.11.6 15G31.  Perhaps an issue with colliding group ownership homebrew sets as well.

**Comment by skurfer** [comment]
Yes, if you just `rm /usr/local/bin/qs` then go back into Quicksilver’s preferences and click the button, it should put the new version in place.

**Comment by rprimus** [comment]
Sat Aug 20 08:42:49 BST 2016

When trying to install the new command line tool, the error  (from ```/var/log/system.log```:

```
Aug 20 08:37:29 mbp Quicksilver[12472]: open on /usr/local/bin/qs: File exists
Aug 20 08:37:49 mbp Quicksilver[12472]: open on /usr/local/bin/qs: File exists
```

**Comment by skurfer** [comment]
This should be fixed in version 1.2.1 of the plug-in.

See https://github.com/quicksilver/CommandLineTool-qsplugin/commit/6c22323453dd402c9456537401f97b67d4ca1a59
